src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -1953,6 +1957,7 @@ await UpdateUsersAsync(group, groupsDict[group.ExternalId].ExternalUserIds,
             }
         }
 
+        await _eventService.LogOrganizationUserEventsAsync(events);
```

Comment:
There's an [overload](https://github.com/bitwarden/server/blob/3d7fe4f8afaa65add13c58c13d4351ae86d0f22c/src/Core/Services/IEventService.cs#L22-L23) that lets you specify an `EventSystemUser`. The idea is that it identifies "who" (or what) made the change when it's not tied to a specific user account - currently just used for SCIM and the domain verification job.

I suggest we add an `EventSystemUser.PublicApi` enum value, so that this displays in the Admin Console logs as a change made via the public api (technically it's not limited to directory connector). Note that this would also require updating the [client-side enum](https://github.com/bitwarden/clients/blob/main/libs/common/src/enums/event-system-user.enum.ts#L2) and [translation mapping](https://github.com/bitwarden/clients/blob/main/apps/web/src/app/admin-console/organizations/manage/events.component.ts#L22-L25) but otherwise should "just work".

Reply:
Implemented as suggested on https://github.com/bitwarden/server/pull/4632/commits/89cdfffde2bbe078e13e20dfeb1f386de85ed34b

Client PR is https://github.com/bitwarden/clients/pull/10660